README.md

```diff
@@ -23,19 +24,39 @@ const urlComponents: URLComponentsType = {
 }
 
 // Create an IrodsHttpClient object
+// <username> and <password> are placeholders
 const api = new IrodsHttpClient(urlComponents, '<username>', '<password>')
 
-let res
+// <username> is the same placeholder value used above
 let lpath = "/tempZone/home/<username>/newCollection"
 
-// Authenticate the client within an asynchronous context.
-// From there, any operations of the api can be done by the client.
-(async () => {
+// After authenticating the client in an asynchronous context, any operations of the api can be done.
+async function authenticateAndCreateCollection(lpath: string) {
     await api.authenticate()
-    res = await api.collections.create({
-        lpath: lpath
-    })
-})
+    let res = await api.collections.create({lpath: lpath})
+    try {
+        // res.status contains the http status code of the response
+        // If res.status >= 400, then res.data is null and there was an error
+        assert(res.status < 400, 'Error obtaining response!')
+
+        // At this point, the response went through, so res.data shouldn't be null
+        // If the irods_response status code is non-zero, there was an error
+        assert(res.data !== null && res.data.irods_response.status_code === 0, 'Error with response data!')
```

Comment:
There are few lines which exceed the width of the visible code area. Break those so the horizontal scrollbar isn't needed.

You can check the rendering by simply looking at this branch in your fork.

Reply:
done